server/container/guice/cassandra-guice/src/test/resources/mailetcontainer.xml

```diff
@@ -63,9 +63,8 @@
                 <spamdHost>localhost</spamdHost>
                 <spamdPort>783</spamdPort>
             </mailet>
-            <mailet match="IsMarkedAsSpam" class="ToRecipientFolder">
+            <mailet match="RecipientIsLocal" class="ToSpamOrInboxFolder">
```

Comment:
I don't understand why you need such a mailet

Reply:
I guess that's because when you have a mail with multirecipents, you need to split it: for some recipients it must go in Spam, for some other in Inbox. I would prefer a more modular solution, but I don't see one. Do you think of something better?

Reply:
The previous version `<mailet match="IsMarkedAsSpam" class="ToRecipientFolder">` worked very well.

I'm also against a `ToSpamOrInboxFolder` that basicaly duplicate `LocalDelivery` job and responsabilities.

If you **really** want a single mailet to do that job, please position storage directives. As a reminder:

```
mail.setAttributte("DeliveryPath_benoit@linagora.com", "Spam");
```

will get Local delivery storing that mail in my **Spam** folder. Maybe this attribute can be set by the SpamAssassin mailet?